### DISCUSSION TITLE
Does promitor support custom dimensions in metrics definitions?

### DISCUSSION THREAD

**Discussion by pikopl** [tomkerkhove/promitor#2543] [Q&A]
Does promitor support custom dimension or override default dimensions for metrics?

I need to use different dimension for EventHub, e.g. OperationResult (according to https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/microsoft-eventhub-namespaces-metrics)
For now, I can see that only [Entity Name](https://docs.promitor.io/v2.11/scraping/providers/event-hubs/) is supported.

Could you help me?

**Comment by tomkerkhove** [comment]
You can only use the dimensions that Azure Monitor metrics provide.

In your case, you can use OperationResult for the metrics listed on https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/microsoft-eventhub-namespaces-metrics
